src/main/app/form/parser.ts

```diff
@@ -28,3 +31,39 @@ export const setupCheckboxParser: CheckboxParser = ([key, field]) => {
   }
   return [key, field];
 };
+
+type StepWithForm = { form?: FormContent } & Step;
+
+export const omitUnreachableAnswers = (caseState: Partial<Case>, steps: Step[]): Partial<Case> => {
+  const sequenceWithForms = (steps as StepWithForm[]).filter(step => step.form);
+
+  const getPossibleFields = (step = sequenceWithForms[0], fields = [] as string[]) => {
```

Comment:
I don't think these arguments need a default value, just add the defaults when call it for the first time

Reply:
👍 Good idea, fix in 0f5e3d9